src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,30 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { body, title, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
```

Comment:
theoretically `user` can be `null`. In this case we will get error (we try to read some properties in `undefined` inside UserInfo).

let's handle this case - do not render `UserInfo` ([conditional rendering](https://react.dev/learn/conditional-rendering#logical-and-operator-)) component at all in this case